test/extern/ExternBlockClangError.chpl

```diff
@@ -1,6 +1,6 @@
 extern {
-  extern void foo();
-  void bar() {
+  extern void foo(void);
+  void bar(void) {
```

Comment:
This test is checking that we get warnings from Clang. I couldn't get clang 14 and 15 to give the same warning for things like `extern void foo()` so opted to have the test focus on just one warning (`no previous prototype for function 'bar'`).